### ISSUE TITLE
Add support for streaming from output cb

### ISSUE THREAD

**Issue by rpavlovicTT** [tenstorrent/tt-mlir#782]
We should add support for streaming results from output cb through data movement kernels to an output tensor.

**Comment by skhatriTT** [comment]
@nsmithtt Any update on this one?

**Comment by nsmithtt** [comment]
No, no plans for the foreseeable future either, I demoted to P2.
